docs/gettingStarted/quickStart.rst

```diff
@@ -185,14 +185,19 @@ Running the example
    back together---is compartmentalized into a **job**. Each job can specify its own resource requirements and will
    only be run after the jobs it depends upon have run. Jobs without dependencies will be run in parallel.
 
-#. Run with custom options::
+.. note::
+        Delete ``fileToSort.txt`` before moving on to #3. The options, ``--numLines`` and ``--lineLength``,
+        specify dimensions for creating ``fileToSort.txt`` if it does not already exist. If it exists, this workflow
+        will use the existing file.
+
+3. Run with custom options::
 
-      (venv) $ python sort.py file:jobStore --numLines=5000 --lineLength=10 --workDir=/tmp/ --overwriteOutput=True
+      (venv) $ python sort.py file:jobStore --numLines=5000 --lineLength=10 --overwriteOutput=True --workDir=/tmp/
 
-   Here we see that we can add our own options to a Toil script. The first two
-   options determine the number of lines and how many characters are in each line.
-   The last option is a built-in Toil option where temporary files unique to a
-   job are kept.
+   Here we see that we can add our own options to a Toil script. As noted above, the first two
+   options determine the number of lines and how many characters are in each line. ``--overwriteOutput`` tells the Toil
```

Comment:
`tells the Toil script` makes it sound like it's a toil option to me.

`--overwriteOutput` is a `sort.py` option.  It should be clarified which options are for toil, and which are for the `sort.py` example.